modules/rosetta/src/test/scala/org/tessellation/rosetta/domain/construction/ConstructionServiceSuite.scala

```diff
@@ -36,6 +39,53 @@ object ConstructionServiceSuite extends MutableIOSuite with Checkers with Transa
       .forAsync[IO](sharedKryoRegistrar)
       .flatMap(kryo => SecurityProvider.forAsync[IO].map(securityProvider => (securityProvider, kryo)))
 
+  def generateTransaction(wantSignedTransaction: Boolean) =
```

Comment:
If I read this correctly, it creates more than one transaction

Reply:
Updated.